packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -295,12 +296,16 @@ export class ConnectionService {
   }
 
   /**
-   * Create a trust ping message for the connection with the specified connection id.
+   * Create an ack using a trust ping message for the connection with the specified connection id.
    *
    * @param connectionId the id of the connection for which to create a trust ping message
+   * @param options optional trust ping options
    * @returns outbound message containing trust ping message
    */
-  public async createTrustPing(connectionId: string): Promise<ConnectionProtocolMsgReturnType<TrustPingMessage>> {
+  public async createAck(
```

Comment:
What is the difference between createAck and createTrustPing?

Reply:
No that's completely fair! I've had my fair share of pedantic things--and I see your point too  :) 
I'll go ahead and update this to reflect your thought process here